src/olympia/abuse/utils.py

```diff
@@ -51,11 +51,34 @@ def __init__(self, decision):
                 f'{self.valid_targets}'
             )
 
+    def log_action(self, activity_log_action, *extra_args, extra_details=None):
+        return log_create(
+            activity_log_action,
+            self.target,
+            *(self.decision.policies.all()),
+            *extra_args,
+            details={
+                'comments': self.decision.notes,
+                'cinder_action': self.decision.action,
+                **(extra_details or {}),
+            },
+        )
+
+    def should_hold_action(self):
```

Comment:
the should_hold_action and hold_action are just empty methods that should be overridden by child classes right? Why not raise a not implemented error?